resource-definitions/terraform-driver/private-git-repo/https-secret-refs.tf

```diff
@@ -25,9 +25,9 @@ resource "humanitec_resource_definition" "example-resource" {
     values_string = jsonencode({
       # Connection information to the target Git repo
       source = {
-        path = "some-resource-type/terraform"
-        rev  = "refs/heads/main"
-        url  = "https://my-domain.com/my-org/my-repo.git"
+        path   = "some-resource-type/terraform"
+        branch = "refs/heads/main"
```

Comment:
For the `k8s-cluster-git` Driver, this does indeed need to be `branch`. But these Resource Definitions are using the Terraform Driver and then `rev` is correct according to our [current documentation](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#source-object). 
Am I confusing something?

Reply:
@TobiasBabin ! ah you are right! sorry ! closing